translate/rus.json

```diff
@@ -400,7 +401,7 @@
 "TCEC draw rule":"правило ничьи TCEC",
 "TCEC is based on voluntary work.":"TCEC основан на добровольной работе.",
 "TCEC is powered by":"TCEC работает на",
-"TCEC is sponsored by [SPONSOR]":"TCEC спонсируется [SPONSOR]ом",
+"TCEC is sponsored by [SPONSOR]":"Спонсоры TCEC: [SPONSOR]",
```

Comment:
This line is shortened and simplified: now you don't need to add anything after a [SPONSOR] since the structure of the sentence is now constructed in such a way that it is no longer required. You can put [SPONSOR1], [SPONSOR2], [SPONSOR3] all in line without even hesitating. I hope you understood me.

Reply:
yeah better